src/nav/mobile-monitoring.yml

```diff
@@ -75,11 +75,12 @@ pages:
                 path: /docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps
               - title: iOS device obfuscation
                 path: /docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-device-id-obfuscation
-
           - title: tvOS
             pages:
               - title: tvOS compatibility and requirements
                 path: /docs/mobile-monitoring/new-relic-mobile-ios/tvos/new-relic-tvos-compatibility-requirements
+          - title: Instrument your watchOS project with New Relic
```

Comment:
```suggestion
          - title: watchOS
```

Reply:
I can make this a watchos folder with watchOS doc in it, similar to tvOS. lmk what you prefer